### DISCUSSION TITLE
Connection to Metamask is unable

### DISCUSSION THREAD

**Discussion by adimiuprix** [ConsenSys-archive/truffle#5339] [General Q&A]
Hello, I'm a Windows user. I'm trying to connect to ganache, when I enter RPC url ganache is fine, but input chain ID doesn't respond correctly.
![Capture](https://user-images.githubusercontent.com/41278399/180205215-0532ac30-fc48-4cd6-8f09-c106458d8bb0.PNG)

Please check, here the status of ganache is working normally, just installed.
Thank you

**Comment by apdan510** [comment]
@Adipiter can you please share the list of all of your available test networks? There might be a duplicate localhost network. Thank you!
